### PR TITLE
feat: teleport home command

### PR DESCRIPTION
**Pull request by 0xD-Fabio**
# What? 
Implements /goto home chat command

# Why? 
...

### CODE REVIEW

File: packages/shared/world/TeleportController.ts [hunk 1]
```diff
@@ -60,6 +66,24 @@ export class TeleportController {
     return TeleportController.goTo(x, y, tpMessage)
   }
 
+  public static async goToHome(): Promise<{ message: string; success: boolean }> {
+    try {
+      let homeCoordinates = await fetchHomePoint()
```
Comment (kuruk-mm) [inline packages/shared/world/TeleportController.ts:71]:
```suggestion
      const homeCoordinates = await fetchHomePoint()
```

File: packages/shared/world/TeleportController.ts [hunk 2]
```diff
@@ -78,6 +102,19 @@ export class TeleportController {
   }
 }
 
+async function fetchHomePoint(): Promise<float[]> {
```
Comment (kuruk-mm) [inline packages/shared/world/TeleportController.ts:105]:
```suggestion
async function fetchHomePoint(): Promise<{ x: number; y: number }> {
```

**Review by kuruk-mm — CHANGES_REQUESTED** [review]

File: packages/shared/world/TeleportController.ts [hunk 3]
```diff
@@ -78,6 +101,19 @@ export class TeleportController {
...
 
+async function fetchHomePoint(): Promise<{ x: number; y: number }> {
+  const homePoint: string = await getFromPersistentStorage(homePointKey)
+  if (homePoint) {
+    store.dispatch(sendHomeScene(homePoint))
+    if (!HAS_INITIAL_POSITION_MARK) {
+      const [x, y] = homePoint.split(',').map((p) => parseFloat(p))
+      gridToWorld(x, y, lastPlayerPosition)
+      return { x, y }
+    }
```
Comment (menduz) [inline packages/shared/world/TeleportController.ts:112]: Is this condition right? why does it has a HAS_INITIAL_POSITION_MARK if?

Comment (0xD-Fabio) [reply packages/shared/world/TeleportController.ts]: In my ignorance, I thought it was needed, sorry.

File: packages/shared/world/TeleportController.ts [hunk 4]
```diff
@@ -90,6 +113,17 @@ export class TeleportController {
   }
 }
 
+async function fetchHomePoint(): Promise<{ x: number; y: number }> {
+  const homePoint: string = await getFromPersistentStorage(homePointKey)
+  if (homePoint) {
+    store.dispatch(sendHomeScene(homePoint))
```
Comment (menduz) [inline packages/shared/world/TeleportController.ts:119]: All code looks good but I don't find a reason for this line
```suggestion
    store.dispatch(sendHomeScene(homePoint))
```

Comment (0xD-Fabio) [reply packages/shared/world/TeleportController.ts:119]: I thought this line was required to send the scene to the unity client, if there is no point in it, let's delete it. Thanks.

**Review by menduz — APPROVED** [review]

**Review by kuruk-mm — DISMISSED** [review]

**Comment by menduz** [comment]
@0xD-Fabio please rebase with signed commits to merge

**Pull request merged**